stream_alert/alert_processor/outputs.py

```diff
@@ -258,19 +256,17 @@ def _check_container_exists(self, rule_name, container_url, headers):
                 will be sent or False if a matching container does not yet exists
         """
         # Limit the query to 1 page, since we only care if one container exists with
-        # this name. This should not be a problem, but utilize urllib.quote to
-        # replace any unsafe characters in the rule_name
-        query_url = '{}?_filter_name="{}"&page_size=1'.format(container_url,
-                                                              urllib.quote(rule_name))
-
-        # Passing None for the data param ensures a GET request happens
-        resp = self._request_helper(query_url, None, headers, False)
-
+        # this name.
+        params = {
+            '_filter_name': rule_name,
+            'page_size': 1
+        }
+        resp = self._get_request(container_url, params, headers, False)
         if not self._check_http_response(resp):
             return False
 
         try:
-            resp_dict = json.loads(resp.read())
+            response = json.loads(resp.text)
```

Comment:
This is no longer necessary with requests (I don't think) and you can just use `resp.json()` without the try/except